data_structures/read_group.wdl

```diff
@@ -63,6 +63,7 @@ struct ReadGroup {
 
 workflow read_group_to_string {
     meta {
+        name: "ReadGroup to String"
```

Comment:
Personally, I don't know that I find the `name` key all that informative. Is it so that `wdldoc` knows what to title the page? I suppose this is easier than parsing `read_group_to_string`.

Reply:
That is much nicer. Agreed that it's easy enough to ignore. I wasn't overly concerned. I wrote the first part of my comment before realizing it was likely for `wdldoc`.